838 - fix parse CRD error: added CRD 1.16+ spec

realshuting opened the pull request:
This PR fixes #838, by parsing the newer version of CRD to a difference JSON object.

Changes:
- increase CRD sync period to 10 minutes
- increase background sync period to 5 minutes
- fix "failed to list resources" for Namespace
- suppress logs

JimBugwadia commented on pkg/openapi/crdSync.go:
```diff
@@ -54,7 +84,7 @@ func (c *crdSync) Run(workers int, stopCh <-chan struct{}) {
 
 	for i := 0; i < workers; i++ {
-		go wait.Until(c.sync, time.Second*25, stopCh)
+		go wait.Until(c.sync, 10*time.Minute, stopCh)
```
Can we pull all constants into a common file? Ideally, we can make these configurable over time.

JimBugwadia commented on pkg/policyviolation/generator.go:
```diff
@@ -153,7 +153,7 @@ func (gen *Generator) Run(workers int, stopCh <-chan struct{}) {
 
 	for i := 0; i < workers; i++ {
-		go wait.Until(gen.runWorker, time.Second, stopCh)
+		go wait.Until(gen.runWorker, 5*time.Minute, stopCh)
```
same comment - extract constants

JimBugwadia commented:
@realshuting - looks like the new constants package is missing..,

realshuting commented:
I was wondering why CI failed.. thanks :)

realshuting commented:
@JimBugwadia 
I rebased the master, not sure why it shows the merged changes. The main change since the last review is added pkg/constant/constant.go.
